### ISSUE TITLE
more than 2 start values issues

### ISSUE THREAD

**Issue by raziza** [Yankovsky/nouislider-angular#30]
When having more than 2 values to our slider only first 2 are being synced.

Fix: 'equals' and `createSlider` methods should be changed to loop over the start values and not hard coded first and second items.

**Comment by Yankovsky** [comment]
Actually, I didn't know that nouislider supports more than 2 handles.

**Comment by Yankovsky** [comment]
Ok, it was added in 9.0.0 version and currently this lib supports ~8.5.1

**Comment by leongersen** [comment]
@Yankovsky While technically not b/w compatible (the DOM changes), upgrading to noUiSlider 9 should be hassle-free. There are no API changes.

**Comment by raziza** [comment]
Yeah, I've changed this part in your code and nothing else and looks like it is working fine (so far).

**Comment by kikitsa** [comment]
Same problem here. As @raziza said you have to change the values in a loop in update function (line 78). In addition you have to write a new equals function (line 14), for example [here](https://stackoverflow.com/questions/4025893/how-to-check-identical-array-in-most-efficient-way)

Are you planning for an updated release?

**Comment by Yankovsky** [comment]
@raziza as I understand you've solved this issue. Can you create pr?

**Comment by Yankovsky** [comment]
@kikitsa @leongersen Do you still need help with that issue?
